s3torchconnectorclient/python/tst/integration/test_logging.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import os.path
+import sys
+from datetime import datetime
+
+import pytest
+
+PYTHON_TEST_CODE = """
+import logging
+import os
+import sys
+
+os.environ["ENABLE_CRT_LOGS"] = "{0}"
+
+logs_dir_path = "{1}"
```

Comment:
Why are we using sys.argv lower down, and string formatting here?

Reply:
Aligning to use sys.argv